docs/Quickstart-Guide-for-Beginners.md

```diff
@@ -124,26 +124,35 @@ startleft parse --help
 ```
 
 ### validate
-This command is able to perform validations over three types of files:
+This command is able to perform validations over several types of files:
 #### **IaC mapping files**
 Described in depth in each processor's docs, they are used to create relationships between types in the source
 and their expected equivalent in the OTM (i.e: an `aws_instance` type in Terraform matches the `ec2` type in IriusRisk).
 If we take the same mapping file we have downloaded for the `parse` command, we can execute:
-```shell
-startleft validate --iac-mapping-file iriusrisk-tf-aws-mapping.yaml
-```
+
+??? example "Terraform example"
+
+    ```shell
+    startleft validate --mapping-file iriusrisk-tf-aws-mapping.yaml --mapping-type TERRAFORM
+    ```
+
+??? example "Cloudformation example"
+
+    ```shell
+    startleft validate --mapping-file iriusrisk-cft-mapping.yaml --mapping-type CLOUDFORMATION
+    ```
 
 #### **Diagram mapping files**
 Also described in the processors' documentation, allows you to validate the format of mapping
-files used for diagram conversions. 
+files used for diagram conversions as Visio and Lucidchart. 
     
 Let's download the IriusRisk's Visio mapping file located in the `examples/visio` folder:
 ```shell
 wget https://raw.githubusercontent.com/iriusrisk/startleft/main/examples/visio/iriusrisk-visio-aws-mapping.yaml
```

Comment:
I would put the same format for all examples, I mean put this kind of drop-down example or just as code.